app/graphicsview.cpp

```diff
@@ -258,22 +258,22 @@ void GraphicsView::dropEvent(QDropEvent *event)
     if (mimeData->hasUrls()) {
         const QList<QUrl> &urls = mimeData->urls();
         if (urls.isEmpty()) {
-            showText(tr("File url list is empty"));
+            showText(tr("The file URI list is empty"));

```

Comment:
Not quite sure about the actual differences between URI and URL, since we (including Qt itself) are using QUrl all the time, maybe keep using "URL" could be better?

Reply:
Ideally none of those, but will make it URL.